qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/create/CreateOffer.groovy

```diff
@@ -27,17 +27,19 @@ class CreateOffer implements CreateOfferInput{
     @Override
     void createOffer(Offer offerContent) {
         OfferId identifier = generateQuotationID(offerContent.customer)
+        double offerPrice = PriceCalculator.calculateOfferPrice(offerContent.items,offerContent.selectedCustomerAffiliation.category)
```

Comment:
I'm a bit confused by the computation of the price here, as well as in the implemented method of the interface.

Reply:
I agree. And the business rules of calculating the price are part of the business object `Offer`